skimage/measure/fit.py

```diff
@@ -785,54 +790,61 @@ def ransac(data, model_class, min_samples, residual_threshold,
 
     random_state = check_random_state(random_state)
 
-    if min_samples < 0:
-        raise ValueError("`min_samples` must be greater than zero")
+    # in case data is not pair of input and output, male it like it
+    if not isinstance(data, (tuple, list)):
+        data = (data, )
+    num_samples = len(data[0])
+
+    if not (0 < min_samples < num_samples):
+        raise ValueError("`min_samples` must be in range (0, <nb-samples>)")
```

Comment:
`nb` is very hard to understand. I know this makes sense to us now, but trust me when I say that people not use to the lingo will not understand very clearly.

Just use `len(data[0])` or `number of samples`.

Reply:
I had there `number-of-samples` then @jni proposed change...
I would not write `data[0]` since it may be confusing if you have jut single data, not input/output data pair...

Reply:
I think it is probably best you change both to be the same thing, `number-of-samples` maybe?

Reply:
I have changed it here, but I would keep the doc notation as N to be consistent with the rest of the text...